packages/jsActions/mobile-resources-native/src/notifications/HasNotificationPermission.ts

```diff
@@ -20,7 +20,7 @@ export async function HasNotificationPermission(): Promise<boolean> {
     }
 
     return NativeModules.RNFBMessagingModule.hasPermission().then((authStatus: number) => {
-        if (authStatus) {
+        if (authStatus && authStatus > 0) {
```

Comment:
RNFBMessagingModule provides `AuthorizationStatus` Enum that we can use here to make code more readable

Reply:
Hi Ahmed, thank you for the review! We're trying to avoid importing RNFBMessagingModule here, because that is currently making the Firebase configuration mandatory in the Native Builder UI. This action is however also used for local notifications, where Firebase is not be required.

Reply:
Hi @Ahmad-Elsayed, thanks a lot for helping out with improving readability this morning. I just committed the updated code. Can you review this again? Thank you in advance!